src/module/token/tokenId/SequentialTokenIdERC1155.sol

```diff
@@ -73,6 +75,29 @@ contract SequentialTokenIdERC1155 is Module, UpdateTokenIdCallbackERC1155 {
         return _tokenId;
     }
 
+    /// @dev Called by a Core into an Module during the installation of the Module.
+    function onInstall(bytes calldata data) external {
+        uint256 nextTokenId = abi.decode(data, (uint256));
+        _tokenIdStorage().nextTokenId = nextTokenId;
+    }
+
+    /// @dev Called by a Core into an Module during the uninstallation of the Module.
+    function onUninstall(bytes calldata data) external {}
+
+    /*//////////////////////////////////////////////////////////////
+                    Encode install / uninstall data
+    //////////////////////////////////////////////////////////////*/
+
+    /// @dev Returns bytes encoded install params, to be sent to `onInstall` function
+    function encodeBytesOnInstall(uint256 nextTokenId) external pure returns (bytes memory) {
```

Comment:
Would call this startTokenId to be clear of what it is

Reply:
addressed